tests/containers/build.pm

```diff
@@ -5,9 +5,9 @@ use testapi;
 sub run {
     my ($self) = @_;
     assert_script_run("git clone https://github.com/os-autoinst/openQA.git", timeout => 300);
-    assert_script_run("docker build openQA/container/webui -t openqa_webui", timeout => 600);
-    assert_script_run("docker build openQA/container/worker -t openqa_worker", timeout => 600);
-    assert_script_run("docker build openQA/container/openqa_data -t openqa_data", timeout => 600);
+    assert_script_run('for i in {1..3}; do docker build openQA/container/webui -t openqa_webui && break; done', timeout => 600);
```

Comment:
You could also use `script_retry` to achieve the same but with a bit better readability:
```suggestion
    script_retry('docker build openQA/container/webui -t openqa_webui', retry => 3, timeout => 600);
    script_retry('docker build openQA/container/worker', retry => 3, timeout => 600);
    script_retry('docker build openQA/container/openqa_data -t openqa_data', retry => 3, timeout => 600);
```

Reply:
And of course `use utils;` at the beginning.

Reply:
@grisu48 not possible, see https://github.com/os-autoinst/os-autoinst-distri-openQA/pull/66#issuecomment-820355298

Reply:
> @grisu48 not possible, see [#66 (comment)](https://github.com/os-autoinst/os-autoinst-distri-openQA/pull/66#issuecomment-820355298)

I saw your comment, but was missing the context, that this is in the openqa and not in the os-autoinst-opensuse framework. Please just ignore my noise.